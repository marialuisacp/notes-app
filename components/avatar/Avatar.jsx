import React from 'react';
import StyledAvatar from './styles/Avatar.styles';

const Avatar = () => (
  <StyledAvatar></StyledAvatar>
);

export default Avatar;