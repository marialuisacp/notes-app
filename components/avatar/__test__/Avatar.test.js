import React from 'react';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import Avatar from '../Avatar';

test('should render component Avatar', () => {
  const { fragment } = render(<Avatar></Avatar>);
  expect(fragment).toMatchSnapshot();
});
