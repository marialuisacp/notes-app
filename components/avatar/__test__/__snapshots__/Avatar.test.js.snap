// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component Avatar 1`] = `undefined`;
